data/reference/syntax.adoc

```diff
@@ -0,0 +1,169 @@
+AsciiDoc Syntax Quick Reference - an overview of the AsciiDoc syntax.
```

Comment:
I have several questions about the formatting.

#### Indentation

Indentations are commonly used to create visual blocks, for instance:

```
Positional Arguments:
  
  These arguments come after any flags and in the order they are listed here.
  Only URL is required.
```

The main issue with indentation in our case is that AsciiDoc syntax is "space sensitive".
So if user copy and paste an example, it will be converted as a literal paragraph:

```
Text formatting:

   *bold text* 
```

So it may be a bit conter intuitive.

##### Spaces

If we still want to use indentation, how many spaces should we use ?
I found CLI where the indentation is 3 spaces and some others CLI where the identation is 2 spaces...

#### Syntax

We could of course use AsciiDoc syntax to describe AsciiDoc syntax but I think it can be confusing, especially with titles:

```
= Section Titles

= Document Title
== Section 1
=== Section 2
```

Reply:
For reference the content is outdated, I'm just concerned about the text formatting for now.

Reply:
.AsciiDoc syntax
```
= AsciiDoc Syntax Quick Reference - an overview of the AsciiDoc syntax.

== Formatted Text

.Bold, Italic, and Monospace

*bold text*
_emphasized text_
`monospaced text`

== Document Links

.External

https://asciidoctor.org - automatic!
https://asciidoctor.org[Asciidoctor]
https://github.com/asciidoctor[Asciidoctor @ *GitHub*]

.Inline anchors

[[bookmark-a]]Inline anchors make arbitrary content referenceable.
[#bookmark-b]#Inline anchors can be applied to a phrase like this one.#
anchor:bookmark-c[]Use a cross reference to link to this location.
[[bookmark-d,last paragraph]]The xreflabel attribute will be used as link text in the cross-reference link.

== Section Titles

.Headings

= Document Title (Level 0)
== Level 1 Section Title
=== Level 2 Section Title
==== Level 3 Section Title
===== Level 4 Section Title
====== Level 5 Section Title
== Another Level 1 Section Title
```

.Spaced
```
AsciiDoc Syntax Quick Reference - an overview of the AsciiDoc syntax.

Formatted Text:

  Bold, Italic, and Monospace

    *bold text*
    _emphasized text_
    `monospaced text`

Document Links:

  External

    https://asciidoctor.org - automatic!
    https://asciidoctor.org[Asciidoctor]
    https://github.com/asciidoctor[Asciidoctor @ *GitHub*]

  Inline anchors

    [[bookmark-a]]Inline anchors make arbitrary content referenceable.
    [#bookmark-b]#Inline anchors can be applied to a phrase like this one.#
    anchor:bookmark-c[]Use a cross reference to link to this location.
    [[bookmark-d,last paragraph]]The xreflabel attribute will be used as link text in the cross-reference link.

Section Titles:

  Headings

    = Document Title (Level 0)
    == Level 1 Section Title
    === Level 2 Section Title
    ==== Level 3 Section Title
    ===== Level 4 Section Title
    ====== Level 5 Section Title
    == Another Level 1 Section Title
```